### PR TITLE
Runtime fields

### PR DESCRIPTION
**Pull request by haskellcamargo**
This pull-request makes `title` and `description` fields dynamic.
Now, instead of `package.json`, they are passed in the configurations, sibling to `params`, and they can be directly translated.

`rung init` is now dead because `rung boilerplate` can do everything as a superset, and generating a JS file in `init` would assume the behavior of `boilerplate`.

### CODE REVIEW

**Comment by haskellcamargo** [comment]
@codecov, STOP!

File: src/build.js [hunk 1]
```diff
@@ -1,38 +1,142 @@
...
+
+/**
+ * Precompiles the locale files, generating a meta file containing the meta
+ *
+ * @param {String[]} files
+ * @return {Promise}
+ */
+function precompileLocales(files) {
+    return resolve(files)
+        .then(filter(test(/^locales\/[a-z]{2}(_[A-Z]{2,3})?\.json$/)))
```
Comment (cuchi) [inline src/build.js]: I think `{2,3}` is only for the language part, the country code is always 2 chars.

You can check it: `cat /usr/share/i18n/SUPPORTED`

File: src/build.js [hunk 2]
```diff
@@ -1,38 +1,142 @@
...
+function analyzeFiles(files) {
     const missingFiles = without(files, requiredFiles);
+
     if (missingFiles.length > 0) {
-        throw new Error(`Missing ${missingFiles} from the project!`);
+        throw new Error(`missing ${missingFiles.join(', ')} from the project`);
+    }
+
+    if (!contains('icon.png', files)) {
+        emitWarning('compiling extension without providing an icon.png file');
```
Comment (cuchi) [inline src/build.js]: Nice :)

File: src/build.js (hunk 1 above)
Comment (haskellcamargo) [inline src/build.js]: But the regex is correct. It matches against `pt` and `pt_BR`, there is no need to call an external process, Unix-only for that. There is a case for the language characters and an optional case for the dialect.

**Pull request merged**